src/frontend/src/pages/FinancePage/ReimbursementRequestForm/ReimbursementRequestForm.tsx

```diff
@@ -157,10 +156,12 @@ const ReimbursementRequestForm: React.FC<ReimbursementRequestFormProps> = ({
     }
   };
 
-  const allWbsElements: {
-    wbsNum: WbsNumber;
-    wbsName: string;
-  }[] = getAllWbsElements(allProjects);
+  const projectDropdown = allProjects.map((proj) => {
```

Comment:
Can you rename this to something like `allProjectWbsElements` to keep it consistent with what was there previously?